proposals/spiffe.adoc

```diff
@@ -0,0 +1,140 @@
+
+* [SIG] Components (link:https://groups.google.com/a/spiffe.io/forum/#!forum/sig-components[website]): The purpose of this Google Group is to discuss items related to the components and APIs tied to SPIFFE's reference implementation (SPIRE) and its architecture. Topics such as role of Node Agent vs. Cluster CA, API semantics, and others serve as good examples of what's to be discussed.
+
+* [SIG] Specification (link:https://groups.google.com/a/spiffe.io/forum/#!forum/sig-specification[website]): The purpose of this Google Group is to discuss items related to the SPIFFE specifications.
+
+* SPIFFE Announce (link:https://groups.google.com/a/spiffe.io/forum/#!forum/announce[website]): The purpose of this Google Group is to share community-wide announcements about SPIFFE and SPIRE.
+
+* Technical Steering Committee (link:https://groups.google.com/a/spiffe.io/forum/#!forum/tsc[website]): This is an ACLd distribution group for communications amongst members of the SPIFFE’s Technical Steering Committee.
+
+*Website*:
+
+SPIFFE’s link:https://www.spiffe.io/[website] is based on GitHub Pages. It primarily serves as a landing page for the project’s primary documents, and mostly redirects to the GitHub repositories.
+
+*Release methodology and mechanics*
+
+SPIRE operates on a 30 to 60-day release cadence, with releases marked with versioned git tags. RC-quality code is periodically tagged off of the master branch before the final release. RC and final releases include binaries for glibc-based Linux platforms. The SPIFFE standards themselves are currently unversioned.
+
+*Social media accounts*:
+
+SPIFFE’s only social media account is on link:https://twitter.com/spiffeio[Twitter].
+
+*Existing sponsorship*:
+
+link:https://www.scytale.io[Scytale, Inc.] and link:https://www.google.com[Google] currently serves as SPIFFE’s primary sponsors.
+
+*Contributor statistics*:
+
+The various SPIFFE projects currently have 16 active contributors from 8 organizations, including Scytale, Twilio, Square, Buoyant.io, and OvrClk. 11 contributors are granted the ability to commit changes across some or all of the codebase.
+
+*External Dependencies*: 
+
+SPIRE has the following build-time dependencies:
+
+* golang (BSD 3-clause)
+* go.uuid (MIT)
+* golang/protobuf (BSD 3-clause)
+* logrus (MIT)
+* go-grpc (Apache 2.0)
+* go-plugin (MPL-2.0)
+* hcl (MPL-2.0)
+* gorm (MIT)
+* gopsutil (BSD 3-clause)
+* go-hclog (MIT)
+* grpc-gateway (BSD 3-clause, Apache 2.0)
+* inflection (MIT)
+* go-bindata (CC0 1.0)
+* go-sqlite3 (MIT)
+* sqlite (public domain) 
+
+As a golang project, SPIRE has no special runtime dependencies.
+
+*Statement on alignment with CNCF mission*:
+
+We believe aligning on a common representation of workload identity, and proscribing best practices for identity issuance and delivery are critical for widespread adoption of cloud-native architectures. SPIFFE provides exactly this capability.
```

Comment:
s/proscribing/promoting/

Reply:
I think it might be a typo for prescribing but yes promoting is better.

Reply:
Will address this. Thanks for identifying it.